packages/xod-fs/src/write.js

```diff
@@ -10,7 +10,12 @@ export const writeFile = curry((outputPath, data, encoding) => new Promise(
     const resolvedPath = resolvePath(outputPath);
     const fstream = fileSave(resolvedPath);
 
-    fstream.write(data, encoding);
+    const dataWithEol = unless(
+      test(/\n$/g),
+      concat(__, EOL)
```

Comment:
I’m pretty sure we should add a strict `\n`, not EOL. Otherwise, it would still produce falsy diffs (of a more strange kind) when one developer is on Windows and another is on Posix. Does it make sense?

Reply:
@nkrkv fixed :)